Теперь расширение можно поставить в фаерфоксе

RomanSh-shell opened the pull request:
Сделано через https://github.com/mozilla/webextension-polyfill

RomanSh-shell committed:
Исправил слои НСПД

RomanSh-shell committed:
Исправлена иконка мапры

RomanSh-shell committed:
Update init.js

RomanSh-shell committed:
Update open.service.js

RomanSh-shell committed:
Исправлен переход на Google Earth

RomanSh-shell committed:
Update readme.md

google-labs-jules[bot] committed:
Add Firefox support using WebExtensions Polyfill

- Added browser-polyfill.js to the project.
- Included the polyfill in the background script using importScripts().
- Added the polyfill to all content scripts in manifest.json.
- Included the polyfill in the options page (index.html).
- Added browser_specific_settings to manifest.json for Firefox.

google-labs-jules[bot] committed:
Fix: Use background.scripts for Firefox compatibility

- Changed manifest.json to use background.scripts instead of background.service_worker for better Firefox compatibility.
- Removed redundant importScripts('browser-polyfill.js') from resources/system.js as the polyfill is now loaded via manifest.

DimaPopov commented:
Не очень понимаю как связана поддержка Firefox и переход в другие картографические сервисы.

RomanSh-shell commented:
@DimaPopov Так уж вышло что забыл отправить пулриквест перехода. Можешь принять его, пока что

google-labs-jules[bot] committed:
Reverted: Use service_worker for background scripts

- I reverted manifest.json to use background.service_worker for Chrome compatibility.
- I re-added importScripts('browser-polyfill.js') to resources/system.js.

google-labs-jules[bot] committed:
Feat: Add dual background definition for Chrome & Firefox

- I've updated manifest.json to include both `background.service_worker` (for Chrome)
  and `background.scripts` (for Firefox).
- This is based on MDN documentation to allow Firefox to load the extension,
  as it does not support `service_worker` in MV3, while Chrome requires it.

RomanSh-shell committed:
Откат правок по переходу
